executable-spec/test/Cardano/Ledger/Spec/STS/Chain/Chain/Properties.hs

```diff
@@ -70,52 +70,97 @@ relevantCasesAreCovered
   = QC.withMaxSuccess 300
   $ STS.Gen.forAllTrace @(CHAIN Mock) @() maxTraceLength ()
   $ \traceSample ->
+        (QC..&&.)
+          (STS.Gen.classifyTraceLength maxTraceLength 10 traceSample)
+        $
         -- traces should be long enough to allow for the stabilization
         -- of events
-        QC.cover 80
+        QC.cover 70
           ( (fromIntegral $ Trace.traceLength traceSample)
             >=
             getMinTraceLength  (Chain.k $ Trace._traceEnv traceSample)
           )
           "Trace length is long enough to allow for the stabilization of events"
           $
-
-        -- 80% of traces should include a 20% percent of update payload
-        QC.cover 80
-          ( (pctUpdatePayload traceSample) >= 15
-          &&
-            (pctUpdatePayload traceSample) <= 25
+--        QC.collect (pctUpdatePayload traceSample) $
+        -- 50% of traces should include a 20% percent of update payload
+        QC.cover 50
+          ( (pctUpdatePayload traceSample) >= 10
+           &&
+            (pctUpdatePayload traceSample) <= 30
           )
           "a reasonable pct of update payload appears in the trace "
           $
+        -- QC.tabulate "Pct of Txs with Update Payload" [( show @Int
+        --                                                $ round @Float
+        --                                                $ pctUpdatePayload traceSample
+        --                                               ) ++ "%"
+        --                                              ]
+        --  $
+
+        -- X% of traces should: should have submitted SIPs
+        QC.cover 80
+          (submittedSIPsExist traceSample)
+          "Submitted SIPs exist"
+          $
+
+        -- X% of traces should: should have revealed SIPs
+        QC.cover 80
+          (revealedSIPsExist traceSample)
+          "Revealed SIPs exist"
+          $
+
+        -- X% of traces should: should have ballots for SIPs
```

Comment:
We need to replace the `X%`s by actual values throughout or rephrase these comments.

Reply:
Done OK